eox_audit_model/decorators.py

```diff
@@ -37,3 +37,43 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+def audit_drf_api(action='', data_filter=None, hidden_fields=None):
+    """This decorator wraps the functionality of audit_method in order to
+    work with django API view methods,also this allows to filter the data that will be
+    stored in the data base.
+
+    Example
+
+    class YourAPIView(APIView):
+
+        @audit_api_wrapper(action='Get my items', data_filter=['username', 'location'])
+        def get(self, request, *args, **kwargs):
+            ...
+    """
+    def decorator(func):  # pylint: disable=missing-docstring
+        @wraps(func)
+        def wrapper(*args, **kwargs):  # pylint: disable=missing-docstring
+            request = args[1]
+            data = request.data if request.data else request.query_params
+
+            audit_data = {
+                key: value
+                for key, value in data.items()
+                if data_filter and key in data_filter
+            }
+            audit_data.update(
+                {key: '*****' for key in data if hidden_fields and key in hidden_fields}
+            )
+
+            @audit_method(action=action)
+            def api_method(audit_data):  # pylint: disable=unused-argument
```

Comment:
@andrey-canon, One thing I noticed late, if you use this wrapper it will always have `api_method` as the method name.
I can add a `method_name` argument to `audit_drf_api` and change the `__name__` (using a rename decorator, probably) property of `api_method`.

Then when you want to audit a view you call:
`@audit_drf_api(action='my_action', ..., method_name='my_api_method')` 

I'm not sure if it's really important or we can simply let everything be `api_method` and use `action` to describe what is being done.